packages/cli/src/credentials/credentials.service.ts

```diff
@@ -8,8 +8,7 @@ import type {
 } from 'n8n-workflow';
 import { CREDENTIAL_EMPTY_VALUE, deepCopy, NodeHelpers } from 'n8n-workflow';
 import { Container } from 'typedi';
-import type { FindManyOptions, FindOptionsWhere } from 'typeorm';
-import { In, Like } from 'typeorm';
+import type { FindOptionsWhere } from 'typeorm';
```

Comment:
Since `CredentialsService.get` is only called from one place in `EECredentialsController`, we could refactor it and remove this `typeorm` import completely.

Reply:
I've been disregarding/deprioritizing type imports since they will be erased during build.

Reply:
valid point about type imports, but we are also trying to abstract all DB calls into repositories, and in this case `EECredentialsController` is leaking that abstraction [here](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/credentials/credentials.controller.ee.ts#L41-L44).
We don't need to make this change in this PR, but sooner or later, I'd prefer to do this.

Reply:
Got it, I'll do this cleanup when I reach this service, prolly similar to what I did in  `EnterpriseWorkflowService`.